packages/renderer/src/lib/image/ImageDetailsCheck.svelte

```diff
@@ -127,7 +131,9 @@ async function handleAbort() {
 </script>
 
 <ProviderResultPage providers={providers} results={results}>
-  <div class="flex flex-row" slot="header-info">
+  <!-- eslint-disable-next-line sonarjs/no-unused-vars -->
```

Comment:
what is considered unused vars ? snippet ?

Reply:
yes it considers that the function headerInfo() `{#snippet headerInfo()}` is unused

Reply:
I don't really know how to proceed. I saw that some other IDEs have bugs opened around the Snippet / unused variables

Reply:
maybe we can open an issue on sonarjs / eslint side ?

Reply:
I am okay with merging it, I understand the issue